src/crimson/os/seastore/lba_manager/btree/btree_lba_manager.cc

```diff
@@ -107,7 +107,16 @@ BtreeLBAManager::get_mapping(
   Transaction &t,
   laddr_t offset)
 {
-  ceph_abort("not implemented");
+  logger().debug("BtreeLBAManager::get_mapping: {}", offset);
+  return get_root(t
+  ).safe_then([this, &t, offset](auto extent) {
```

Comment:
nit, might want to add a space between `]` and `(`. to be more consistent.

Reply:
fixed.